packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -419,6 +432,9 @@ class _AppBarState extends State<AppBar> {
       appBarIconTheme = appBarIconTheme.copyWith(
         opacity: opacity * (appBarIconTheme.opacity ?? 1.0)
       );
+      appBarActionsIconTheme = appBarActionsIconTheme.copyWith(
```

Comment:
Is this necessary because the icons aren't children of the toolbar?

Reply:
I believe this is needed so that the icons match the toolbar opacity